spec/presenters/sufia/file_set_presenter_spec.rb

```diff
@@ -4,7 +4,16 @@
   let(:solr_document) { SolrDocument.new(file.to_solr) }
   let(:ability) { double "Ability" }
   let(:presenter) { described_class.new(solr_document, ability) }
-  let(:file) { build(:file_set).tap { |f| f.apply_depositor_metadata(user) } }
+  let(:file) do
+    build(:file_set, id: '123abc').tap do |f|
```

Comment:
You can supply `user: user` as a parameter and then drop the block.  It needs to respond to `user_key` though, so `user: double(user_key: 'sarah')` should work.